src/_includes/meta.html

```diff
@@ -1,17 +1,42 @@
-{% if include.title %} {% assign title=include.title|prepend=" - " %} {% else %} {% assign title="" %} {% endif %}
+{%- if page.title -%}
+  {%- assign page_title = page.title -%}
+  {%- assign title = page.title | append: " - " | append: site.title -%}
+{%- else -%}
+  {%- assign title = site.title -%}
+{%- endif -%}
+
+{%- if page.description -%}
+  {%- assign description = page.description | strip_newlines -%}
+{%- elsif page.intro -%}
+  {%- assign description = page.intro | strip_newlines -%}
+{%- else -%}
+  {%- assign description = site.description | strip_newlines -%}
+{%- endif -%}
 
 <meta charset="utf-8" />
-<title>{{site.title}}{{title}}</title>
-<meta name="description" content="{{site.description}}" />
+<title>{{ title }}</title>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 <meta name="viewport" content="width=device-width, initial-scale=1" />
-
-<meta property="og:title" content="{{site.title}}" />
-<meta property="og:type" content="website" />
-<meta property="og:url" content="{{site.url}}" />
-<meta property="og:image" content="{{site.url}}/images/og-image.png" />
+<meta name="description" content="{{ description }}" />
+{%- if page.date and page.tags -%}
+  {%- for tag in page.tags -%}
+    <meta property="article:tag" content="{{ tag }}" />
+  {%- endfor -%}
+  <meta property="article:published_time" content="{{ page.date | date: "%FT%T%z" }}" />
+  <meta property="og:type" content="article">
+{%- else -%}
+  <meta property="og:type" content="website">
+{%- endif -%}
+<meta property="og:locale" content="en_US">
+<meta property="og:site_name" content="{{ site.title }}">
+<meta property="og:title" content="{{ page_title }}" />
+<meta property="og:url" content="{{ page.url | absolute_url }}" />
+<meta property="og:image" content="{{ site.url | absolute_url }}/images/og-image.png" />
 <meta name="twitter:card" content="summary_large_image" />
-<meta property="twitter:domain" content="{{site.domain}}" />
-<meta property="twitter:url" content="{{site.url}}" />
-<meta name="twitter:title" content="{{site.title}}" />
-<meta name="twitter:description" content="{{site.description}}" />
-<meta name="twitter:image" content="{{site.url}}/images/og-image.png" />
+<meta name="twitter:creator" content="@California_ITP">
+<meta name="twitter:description" content="{{description}}" />
+<meta name="twitter:image" content="{{ site.url | absolute_url }}/images/og-image.png" />
+<meta name="twitter:site" content="@California_ITP">
+<meta name="twitter:title" content="{{ page_title }}" />
```

Comment:
Same here - do we need an `if page_title` around this?

Reply:
Good catch. I set `page_title = site.title` if we don't have a `page.title`